src/tail/tail.rs

```diff
@@ -187,7 +187,7 @@ pub fn uumain(args: Vec<String>) -> i32 {
 
     let files = given_options.free;
 
-    if files.is_empty() {
+    if files.is_empty() || &files[0] == "-" {
```

Comment:
I'm fairly positive that `-` can be anywhere in the list of files.

Reply:
Looking at [the code](https://github.com/coreutils/coreutils/blob/master/src/tail.c#L2394) you appear to be right. I guess something like this would work:
```suggestion
    if files.is_empty() || files.iter().any(|&x| x == "-") {
```

Reply:
Not quite, `-` should be treated as a file and not override other file arguments.
for example: 
```sh
$ tail my_file -
==> my_file <==
<file_content>

==> standard input <==
```